webpack.config.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
I have updated to the version. The license were auto-generated, but I have updated them now.